mantidproject/mantid
# Indirect Data Reduction ISIS Calibration - update colour of range bars

Range bars colours in the ISIS Calibration interface have been updated to match the convention in the fit wizard

**To test:**
- Switch default facility/instrument to ISIS/IRIS (View > Preferences > Mantid)
- Open ISIS Calibration (Interfaces > Indirect > Data Reduction > ISIS Calibration)
- RunNo = 26176
- Check the `Create RES file` option 
- Ensure the interface looks like the following where background is blue and Peak/Rebinning is red

![image](https://cloud.githubusercontent.com/assets/13132128/16076930/b225127c-32ed-11e6-96f4-30385ec4054b.png)

Fixes #16491

[RELEASE NOTES](https://github.com/mantidproject/mantid/blob/5489fdf0c698b79fc11d0d438e55e3f2e629c17f/docs/source/release/v3.8.0/indirect_inelastic.rst#improvements)

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
